orc8r/cloud/go/services/state/indexer/reindex/queue.go

```diff
@@ -55,6 +55,9 @@ type JobInfo struct {
 // State indexers are added to the queue for reindexing at initialization, and removed from the queue after the reindex is complete.
 // ClaimAvailableJob should be polled periodically, as jobs may become available at any time.
 type JobQueue interface {
+	// Tracks version info for all tracked indexers
```

Comment:
General nit: prepend with identifier name
```suggestion
	// Versioner tracks version info for all tracked indexers
```

Reply:
Nit: we would want this to be `// Versioner tracks version info...`